web/component/ads/view.jsx

```diff
@@ -121,7 +121,7 @@ function Ads(props: Props) {
     return (
       <div
         className={classnames('ads ads__claim-item', className, {
-          'ads__claim-item--tile': tileLayout,
+          'ads__claim-item--tile': tileLayout, // with no tileLayout it indicates sidebar ad
```

Comment:
This comment is wrong.  Ads in Category Page and Channel Page can be either Tile or List layout.
This ad component is general purpose and not tied to sidebar.